x-pack/plugins/reporting/server/usage/get_export_stats.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DEPRECATED_JOB_TYPES } from '../../common/constants';
```

Comment:
This file was rewritten and renamed: `decorate_range_stats.ts → ...ing/server/usage/get_export_stats.ts`

Previously, this code had a tangle of type ambiguity in the `getAvailableTotalForFeature` function